ext/st/README.md

```diff
@@ -1,20 +1,25 @@
-## STM32 device headers
+## CMSIS device headers for STM32
 
 This folder includes the CMSIS device headers for all STM32 devices.
 The files located here are part of the `STM32Cube` libraries and can be found inside the `STM32Cube*/Drivers/CMSIS/Device/ST/STM32F*xx/Include` folder.  
-The files are not modified, only copied.
 
-Here is a list of the current Cube version, (device header version in braces) and publishing date of the device headers:
+The files are copied and modified by converting all line endings from Windows to Unix style and removing all trailing whitespace (see `post_script.sh`). If you don't like that, stop complaining and do the work yourself.
 
-- [CubeF0 v1.6.0 (v2.3.0) @ 27-May-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef0.html)
-- [CubeF1 v1.4.0 (v4.1.0) @ 29-April-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef1.html)
-- [CubeF3 v1.5.0 (v2.3.0) @ 29-April-2015](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef3.html)
-- [CubeF4 v1.12.0 (v2.5.0) @ 22-April-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef4.html)
-- [CubeF7 v1.4.0 (v1.1.0) @ 22-April-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef7.html)
+Here is the list of the current Cube version, device header version in braces and release date of the device headers:
+
+- [CubeF0 v1.7.0 (v2.3.1) released 04-November-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef0.html)
+- [CubeF1 v1.4.0 (v4.1.0) released 29-April-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef1.html)
+- [CubeF2 v1.4.0 (v2.1.2) released 29-June-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef2.html)
+- [CubeF3 v1.7.0 (v2.3.1) released 16-December-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef3.html)
+- [CubeF4 v1.14.0 (v2.5.0) released 22-April-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef4.html)
```

Comment:
All headers are updated to the latest version, will reimport in a second.
[cmsis-header-stm32](https://github.com/modm-io/cmsis-header-stm32) now checks daily for new versions thanks to Travis CI! [Check it out!](https://travis-ci.org/modm-io/cmsis-header-stm32/builds/212266527)